backend/routers/data.py

```
@@ -128,31 +128,34 @@ def use_sample_data():
             request.url, request.max_pages
         )
 
-        if not reviews:
+        reviews = result["reviews"]
+        total_count = result["total_count"]
+
+        if total_count == 0:
             raise HTTPException(
                 400,
                 "리뷰를 찾을 수 없습니다. URL을 확인해주세요.",
             )
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)